.github/workflows/on-schedule.yml

```diff
@@ -0,0 +1,79 @@
+name: Scheduled Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  test-with-live-bky-server:
+    name: Test Attestation Service Examples Against Live BKY-AS Server
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write # Required for OIDC
+      contents: read  # Required to clone repo
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4.1.0
+        with:
+          # Use the GH Actions role with limited permissions (i.e. read only access)
+          # defined here https://github.com/blocky/atlantis-aws/blob/main/aws_iam.tf
```

Comment:
this is outdated, I think it's supposed to point to this: https://github.com/blocky/atlantis-aws/blob/main/aws_iam_role.tf

This applies to the comment in the on-pr workflow as well